x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings/use_latest_findings.ts

```diff
@@ -63,42 +70,52 @@ export const getFindingsQuery = ({
   sort,
   pitId,
 }: UseFindingsOptions & { pitId: string }) => ({
-  query,
-  size,
-  from,
-  sort: [{ [getSortKey(sort.field)]: sort.direction }],
+  body: {
+    query,
+    sort: [{ [getSortKey(sort.field)]: sort.direction }],
+    size,
+    from,
+    aggs: { count: { terms: { field: 'result.evaluation.keyword' } } },
+  },
   pit: { id: pitId },
   ignore_unavailable: false,
 });
 
-export const useLatestFindings = ({ query, sort, from, size }: UseFindingsOptions) => {
+export const useLatestFindings = (options: UseFindingsOptions) => {
   const {
     data,
     notifications: { toasts },
   } = useKibana().services;
   const { pitIdRef, setPitId } = useContext(FindingsEsPitContext);
   const pitId = pitIdRef.current;
 
-  return useQuery<
-    IEsSearchResponse<CspFinding>,
-    unknown,
-    CspFindingsQueryData & { newPitId: string }
-  >(
-    ['csp_findings', { query, sort, from, size, pitId }],
-    () =>
-      lastValueFrom<IEsSearchResponse<CspFinding>>(
-        data.search.search({
-          params: getFindingsQuery({ query, sort, from, size, pitId }),
+  return useQuery(
+    ['csp_findings', { options }],
```

Comment:
added it back, although i'm not sure where the user will get a different `pit` as unlike `options` this isn't provided by the search bar / table. 

also we're setting a new `queryKey` in `onSuccess`, without a `ref` this will keep triggering a request. initially removed as i figured it's not needed and may be a problem later. but probably too caution..